### PR TITLE
chore(site): refactor stories and tests from components directory

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
- Migrate stories to use non-deprecated Storybook v7 API
- Remove render tests from the integration tests layer
- Remove unnecessary or duplicated stories

### CODE REVIEW

File: site/src/components/CodeExample/CodeExample.tsx [hunk 1]
```diff
@@ -48,14 +39,14 @@ const useStyles = makeStyles<Theme, styleProps>((theme) => ({
     display: props.inline ? "inline-flex" : "flex",
     flexDirection: "row",
     alignItems: "center",
-    background: props.inline ? "rgb(0 0 0 / 30%)" : "hsl(223, 27%, 3%)",
-    border: props.inline ? undefined : `1px solid ${theme.palette.divider}`,
+    background: "rgb(0 0 0 / 30%)",
```
Comment (mtojek) [inline site/src/components/CodeExample/CodeExample.tsx:42]: just double-checking: is it intended? `hsl(223, 27%, 3%)` disappeared.

File: site/src/components/Resources/ResourceAvatar.stories.tsx [hunk 2]
```diff
@@ -1,61 +1,66 @@
...
-export const UnknownResource = Template.bind({});
-UnknownResource.args = {
-  resource: {
-    ...MockWorkspaceResource,
-    type: "noexistentvalue",
+export const UnknownResource: Story = {
+  args: {
+    resource: {
+      ...MockWorkspaceResource,
+      type: "noexistentvalue",
```
Comment (mtojek) [inline site/src/components/Resources/ResourceAvatar.stories.tsx:53]: nit: notexistingvalue or nonexisting

**Review by mtojek — APPROVED** [review]
LGTM

File: site/src/components/CodeExample/CodeExample.tsx [hunk 3]
```diff
@@ -48,14 +39,14 @@ const useStyles = makeStyles<Theme, styleProps>((theme) => ({
...
     color: theme.palette.primary.contrastText,
     fontFamily: MONOSPACE_FONT_FAMILY,
     fontSize: 14,
     borderRadius: theme.shape.borderRadius,
     padding: theme.spacing(1),
     lineHeight: "150%",
+    border: `1px solid ${theme.palette.divider}`,
```
Comment (BrunoQuaresma) [inline site/src/components/CodeExample/CodeExample.tsx:49]: @mtojek it was added here

**Pull request merged**